tests/rptest/tests/compaction_end_to_end_test.py

```diff
@@ -55,9 +67,12 @@ def topic_segments(self):
     @matrix(key_set_cardinality=[10, 1000, 10000],
```

Comment:
the test are failing because we run out of disk space, maybe we can reduce the number of combinations by using only one key set cardinality ?

Reply:
Ya, I retained [10] so that we have more duplicates in the test cases. Wonder why it only fails in release though..